connect.php

```diff
@@ -0,0 +1,4 @@
+<?php
+$con = mysqli_connect("localhost","jeinglis","cavall01","Login_Info") or die("Some error occurred during connection " . mysqli_error($con));  

```

Comment:
I think the error message could be better, `"Failed to connect to database, check connect.php`?